modules/eventing/pages/eventing-advanced-keyspace-accessors.adoc

```diff
@@ -320,7 +320,7 @@ This API indicates errors via the error object in the return value. Exceptions a
 *The Advanced UPSERT operation*: _result = couchbase.upsert(binding, meta, doc)_
 
 This operation allows updating an existing document in the bucket, or if absent, creating a fresh document with the specified key.
-The operation does not allow specifying CAS (it will be silently ignored).
+Setting CAS is a no-op for `upsert` operation.
```

Comment:
Not sure if this actually is easier to understand. Used the wording from Couchbase Server 7.6.

```suggestion
The operation does not allow you to specify CAS.
```